src/vivarium_public_health/plugins/parser.py

```diff
@@ -85,93 +82,67 @@ def parse_component_config(self, component_config: ConfigTree) -> List[Component
                     )
 
         if "causes" in component_config:
-            self.mark_multi_transition_states(component_config)
-            self.add_default_config_layer(component_config)
-            components += self.get_cause_model_components(component_config["causes"])
-
-        # Create a copy of the config tree excluding "external_config" and
-        # "causes" keys.
-        component_config_dict = component_config.to_dict()
-        component_config_dict.pop("external_configuration", None)
-        component_config_dict.pop("causes", None)
-
-        components += self.process_level(component_config_dict, [])
-        return components
-
-    ##################################
-    # Configuration creation methods #
-    ##################################
-
-    @staticmethod
-    def mark_multi_transition_states(component_config: ConfigTree) -> None:
```

Comment:
Are we dropping this feature altogether or is it implemented elsewhere now?

Reply:
I'm dropping this entirely, as I don't think it has any reason to exist now that we have individual clocks.